mep.mjs

```diff
@@ -0,0 +1,140 @@
+import { readFileSync, readdirSync, statSync } from 'fs';
+import { join } from 'path';
+import yaml from 'js-yaml';
+
+function getTableData(meps = 'meps') {
+  let table = [];
+
+  const readDirectoryRecursively = (directory) => {
+    let results = [];
+    const list = readdirSync(directory);
+
+    list.forEach((file) => {
+      const filePath = join(directory, file);
+      const stat = statSync(filePath);
+
+      if (stat && stat.isDirectory()) {
+        results = results.concat(readDirectoryRecursively(filePath));
+      } else {
+        results.push(filePath);
+      }
+    });
+
+    return results;
+  };
+
+  const mepFiles = readDirectoryRecursively(meps).filter((file) => file.includes('mep-'));
+
+  mepFiles.forEach((imep) => {
+    const text = readFileSync(imep, 'utf-8');
+    const data = text.split('---')[1].trim();
+    const parsedYaml = yaml.load(data);
```

Comment:
This made me think something like the following would be useful:

- https://github.com/jupyter-book/mystmd/issues/1547